tinder.js

```diff
@@ -339,6 +365,28 @@ function TinderClient() {
       makeTinderCallback(callback));
   };
 
+
+  /**
+   * Post new user picture
+   * @param {Function} callback the callback to invoke when the request completes
+   */
+  this.uploadPicture = function(file, callback) {
+    tinderPost('image?',
+      null,
```

Comment:
Looks like two attributes should be populated here: userId (current user ID), and file (the image).